### PR TITLE
Add cloudbuild prod file

### PR DESCRIPTION
**Pull request by eduartua**

### CODE REVIEW

File: cloudbuild.prod.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  _ENVIRONMENT: prod
+  _GCLOUD_PROJECT: etcd-io
+  _HELM_CHART_GIT_REPO: https://github.com/etcd-io/discovery.etcd.io.git
+  _HELM_CHART_GIT_BRANCH: master
+  _HELM_CHART_PATH: kubernetes/helm
+  _K8S_NAMESPACE: discoveryserver
+  _REPO_NAME: discoveryserver
+
+steps:
+- name: 'gcr.io/cloud-builders/git'  # Clone infrastructure repo
```
Comment (victortrac) [inline cloudbuild.prod.yaml]: > \# Clone _Helm Chart_ repo where chart lives

File: cloudbuild.prod.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  args: ['checkout', '${_HELM_CHART_GIT_BRANCH}']
+  dir: '/helm'
+  volumes:
+    - name: 'helm'
+      path: '/helm'
+- name: 'gcr.io/cloudkite-public/gcloud-helm3:20191117'  # Deploy helm chart
+  args:
+  - /usr/local/bin/helm
+  - upgrade
+  - --wait
```
Comment (victortrac) [inline cloudbuild.prod.yaml:31]: Should we add `--atomic` ?

**Review by victortrac — CHANGES_REQUESTED** [review]
I'd like to see the two changes in the comments.

File: cloudbuild.prod.yaml (hunk 2 above)
Comment (eduartua) [reply cloudbuild.prod.yaml:31]: Yes, let's add it. There are potential benefits having it.

**Review by victortrac — APPROVED** [review]

**Pull request merged**